build_projects/dotnet-cli-build/project.json

```diff
@@ -13,15 +13,27 @@
     "System.Xml.XmlSerializer": "4.0.11-rc2-24027",
     "WindowsAzure.Storage": "6.2.2-preview",
 
-    "Microsoft.DotNet.Cli.Build.Framework": {"target":"project"},
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
```

Comment:
Same here, this is from experimentation when I couldn't get P2Ps working in VS. These weird changes will get reverted.